src/plugins_types.h

```diff
@@ -77,19 +80,23 @@ struct lysp_module;
  * required format via ::ly_type_print_clb implementation.
  */
 
-/**
- * @brief Helper function for various plugin functions to generate error information structure.
+/*
+ * @brief Create and fill error structure.
+ *
+ * Helper function for various plugin functions to generate error information structure.
  *
- * @param[in] level Error level of the error.
- * @param[in] code Code of the error.
+ * @param[out] err Pointer to store a new error structure filled according to the input parameters.
```

Comment:
This parameter is in,out. It can be seen in file src/log.c on line 136 and 138-139

Reply:
right, fixed and description extended